### DISCUSSION TITLE
Callout component has no background for "new" variation

### DISCUSSION THREAD

**Discussion by alberto-twilio** [twilio-labs/paste#3102] [General] [answered]
Hello Team!

I am working on a screen that should use the Paste [Callout component](https://paste.twilio.design/components/callout), and something strange is happening: when using the **new** variation, the component has no background at all, is not happening for any other variation which are displayed without any issue.

<img width="950" alt="image (5)" src="https://user-images.githubusercontent.com/109074321/225970137-1195e529-3e56-4827-a3e1-081613f5f7b8.png">

I don’t even see the background color property being defined for it in the developer tools.

<img width="194" alt="image (6)" src="https://user-images.githubusercontent.com/109074321/225970142-73718da1-f019-4778-8faa-df60b652459d.png">

The component was not previously used so I installed it, the version I am using is 2.0.0, also tried 1.0.2 but the look is different.

Am I missing something?

Thanks in advance for the support!

**Comment by loreina** [comment]
Hi @alberto-twilio, can you share a reproduction of the issue in CodeSandbox? Thanks!

**Reply by alberto-twilio** [reply]
https://codesandbox.io/s/callout-has-no-background-cfwnxj?file=/src/index.tsx

**Comment by SiTaggart** [comment, marked as answer]
Hi @alberto-twilio 

This is happening because of a version mismatch. As with any NPM package, if you install a new package you have to make sure you are meeting the requirements of that package, including making sure you are installing the supported versions of it's peer dependencies.

As you can see from the package.json file of Callout v2, it has a list of required peer dependencies and version numbers. https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/callout/package.json

```
"@twilio-paste/design-tokens": "^9.0.0",
"@twilio-paste/icons": "^10.0.0"
```

Your package manager most likely is informing you of unmet peer dependencies requirements as you are running a version of design tokens and icons that does not satisfy the required version number needed by callout.

All in all, I would not recommend individual packages unless you are very confident with NPM package resolutions. The easiest thing to do it just to install Paste core and Paste Icons only, and keep those up to date.
